benches/raster.rs

```diff
@@ -13,17 +13,48 @@
 // limitations under the License.
 
 #![feature(test)]
-extern crate test;
 extern crate font_rs;
+extern crate test;
 
-use test::Bencher;
-use font_rs::raster::*;
 use font_rs::geom::Point;
+use font_rs::raster::*;
+use test::Bencher;
 
 fn draw_shape(r: &mut Raster, s: f32) {
-    r.draw_line(&Point{x:s*10.0, y:s*10.5}, &Point{x: s*20.0, y: s*150.0});
-    r.draw_line(&Point{x:s*20.0, y:s*150.0}, &Point{x: s*50.0, y: s*139.0});
-    r.draw_quad(&Point{x:s*50.0, y:s*139.0}, &Point{x: s*100.0, y: s*60.0}, &Point{x: s*10.0, y: s*10.5});
+    r.draw_line(
+        &Point {
```

Comment:
Not a super big fan of how vertically verbose this gets. One thing that might be a solution is to let draw_line and friends take an `Into<Point>` and let simple float tuples convert. But that's probably orthogonal to this PR.

Reply:
Yah, I didn't like it much either but the maintainer of this repo has a reputation of not being picky. 😜

I might see if I can't tweak the settings to not blow those out so much.